### PR TITLE
fix: add param for multi-label per user's feedback

### PR DESCRIPTION
**Pull request by andrewferlitsch**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/model-builder/create_training_pipeline_image_classification_sample.py [hunk 1]
```diff
@@ -33,7 +34,10 @@ def create_training_pipeline_image_classification_sample(
 ):
     aiplatform.init(project=project, location=location)
 
-    job = aiplatform.AutoMLImageTrainingJob(display_name=display_name)
+    job = aiplatform.AutoMLImageTrainingJob(display_name=display_name,
+                                            prediction_type='classification',
+                                            multi_label=multi_label
+                                           )
```
Comment (dandhlee) [inline samples/model-builder/create_training_pipeline_image_classification_sample.py:40]: nit: re-indent them (lint might make you change this anyway) like the code below on line 44:

```
model = job.run(
        dataset=my_image_ds,
        model_display_name=model_display_name,
        training_fraction_split=training_fraction_split,
        validation_fraction_split=validation_fraction_split,
        test_fraction_split=test_fraction_split,
        budget_milli_node_hours=budget_milli_node_hours,
        disable_early_stopping=disable_early_stopping,
        sync=sync,
    )
```

Comment (andrewferlitsch) [reply samples/model-builder/create_training_pipeline_image_classification_sample.py]: fixed indentation

File: samples/model-builder/create_training_pipeline_image_classification_sample.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ def create_training_pipeline_image_classification_sample(
     display_name: str,
     dataset_id: int,
     model_display_name: Optional[str] = None,
+    multi_label: bool = False,
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply samples/model-builder/create_training_pipeline_image_classification_sample.py:27]: updated the assert in the test.

[the remark this answers is not in the document]
Comment (dandhlee) [reply samples/model-builder/create_training_pipeline_image_classification_sample.py:27]: Tests are successful!

File: samples/model-builder/create_training_pipeline_image_classification_sample_test.py [hunk 3]
```diff
@@ -44,7 +44,9 @@ def test_create_training_pipeline_image_classification_sample(
         project=constants.PROJECT, location=constants.LOCATION
     )
     mock_get_automl_image_training_job.assert_called_once_with(
-        display_name=constants.DISPLAY_NAME
+        display_name=constants.DISPLAY_NAME,
+        multi_label=False,
```
Comment (dandhlee) [inline samples/model-builder/create_training_pipeline_image_classification_sample_test.py:48]: lint: complaining about trailing whitespace
```suggestion
        multi_label=False,
```

Comment (andrewferlitsch) [reply samples/model-builder/create_training_pipeline_image_classification_sample_test.py]: fixed

**Review by dandhlee — APPROVED** [review]
LGTM!

**Review by morgandu — APPROVED** [review]

**Pull request merged**